test/parallel/test-fs-promises.js

```diff
@@ -250,6 +266,24 @@ function verifyStatObject(stat) {
       assert(stats.isDirectory());
     }
 
+    // mkdirp require recursive option to be a boolean.
+    // Anything else generates an error.
+    {
+      const dir = path.join(tmpDir, nextdir(), nextdir());
+      ['', 1, {}, [], null, Symbol('test'), () => {}].forEach((recursive) => {
+        assert.rejects(
+          // mkdir() expects to get a boolean value for options.recursive.
+          async () => mkdir(dir, { recursive: i }),
```

Comment:
Small oversight: it should be `recursive` without `: i`

Reply:
Thank you for your review. I fixed this.